src/web/app/pages-session/session-submission-page/__snapshots__/session-submission-page.component.spec.ts.snap

```diff
@@ -1504,7 +1504,7 @@ exports[`SessionSubmissionPageComponent should snap with feedback session questi
                   ngbtooltip="You can save your responses for this question at any time and come back later to continue."
                   type="submit"
                 >
-                  Submit Responses
+                   Submit Response for Question 4
```

Comment:
Can this extra space be removed for all the new snapshot tests?

Reply:
@NicolasCwy How do I do that? Aren't the snapshots auto-generated?

Reply:
Yup thought that this was caused by your editors formatting settings, but as noted in my other comment it is an actual error